plugins/modules/container_distribution.py

```diff
@@ -109,6 +114,7 @@ def main():
             repository=dict(),
             version=dict(type="int"),
             content_guard=dict(),
+            private=dict(type='bool', default=False),
```

Comment:
Actually, i would not default it here. If you leave it optional, then "not specifying it" equals "not touching it".

Reply:
Makes sense.